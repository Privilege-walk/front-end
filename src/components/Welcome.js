import React from "react";
import "./App.css";

export default function Welcome() {
  return (
    <div className="welcome">
        Welcome to the App!
    </div>
  );
}