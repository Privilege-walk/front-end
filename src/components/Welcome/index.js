import React from "react";
import Typography from '@mui/material/Typography';
import Grid from '@mui/material/Grid';
import Button from '@mui/material/Button';
import Container from '@mui/material/Container';

export default function Welcome(){
    
    return (
        <Container sx={{px : 6, mt: 5}}>
            <Grid container justifyContent='center' alignItems='center' direction='column' spacing={2}>
                <Grid item>
                    <Typography variant="h5" component="div">
                        Perspective Walk
                    </Typography>
                </Grid>

                <Grid item>
                    <Typography variant="p" component="div">
                    Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
                    Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, 
                    when an unknown printer took a galley of type and scrambled it to make a type 
                    specimen book. It has survived not only five centuries, but also the leap into 
                    electronic typesetting, remaining essentially unchanged. It was popularised in the 
                    1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more 
                    recently with desktop publishing software like Aldus PageMaker including versions of 
                    Lorem Ipsum
                    </Typography>
                </Grid>

                <Grid item sx={{ mt: 2 }}>
                    <Button variant="outlined">Start</Button>
                </Grid>
            </Grid>
        </Container>
    )
}