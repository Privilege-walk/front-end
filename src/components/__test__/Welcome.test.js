import React from "react";
import { render } from '@testing-library/react';
import Welcome from '../Welcome';

test("Welcome page renders without crashing", async () => {
    const { asFragment } = render(<Welcome />);
    expect(asFragment()).toMatchSnapshot();
})