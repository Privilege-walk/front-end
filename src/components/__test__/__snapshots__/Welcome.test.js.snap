// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome page renders without crashing 1`] = `
<DocumentFragment>
  <div
    class="welcome"
  >
    Welcome to the App!
  </div>
</DocumentFragment>
`;
